chain/client/cosmos.go

```diff
@@ -44,9 +50,32 @@ type CosmosClient interface {
 func NewCosmosClient(
 	ctx client.Context,
 	protoAddr string,
+	serverTlsCert string,
```

Comment:
Please use `cosmosClientOption` instead to supply `RootCAs`
See the explanation below.